libs/components/src/lib/badge/badge.scss

```
@@ -24,6 +27,7 @@
 	#{variables.$badge-block-size}: #{size.$vvd-size-ultra-condensed};
 
 	display: inline-flex;
+	width: 100%;
```

```suggestion
	inline-size: 100%;
```
it might cause you stylelint issue